.github/workflows/ci.yml

```diff
@@ -76,3 +81,38 @@ jobs:
           slab-url: ${{ secrets.SLAB_BASE_URL_PRE_PROD }}
           job-secret: ${{ secrets.JOB_SECRET }}
           label: ${{ steps.test-start.outputs.label }}
+
+  test-action-start-stop:
+    name: GitHub Actions Test (start -> stop)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871
+
+      - name: Test start instance
+        id: test-start
+        uses: ./
+        with:
+          mode: start
+          github-token: ${{ secrets.SLAB_ACTION_TOKEN }}
+          slab-url: ${{ secrets.SLAB_BASE_URL_PRE_PROD }}
+          job-secret: ${{ secrets.JOB_SECRET }}
+          backend: aws
+          profile: ci-test-start-stop
+
+      - name: Test runner is alive
+        uses: ./.github/workflows/registered_runner.yml
+        with:
+          runner-name: ci-persistent-runner
+
+      - name: Test stop instance
+        id: test-stop
+        if: ${{ always() }}
+        uses: ./
+        with:
+          mode: stop
+          github-token: ${{ secrets.SLAB_ACTION_TOKEN }}
+          slab-url: ${{ secrets.SLAB_BASE_URL_PRE_PROD }}
+          job-secret: ${{ secrets.JOB_SECRET }}
+          label: ci-persistent-runner
```

Comment:
any way to check the runner is still there even if offline and instance did not disappear ?

Reply:
I can add another sub workflow to do that.

Reply:
as you wish

Reply:
or run the workflow twice ?